common/engine_legacy_conf/service.cc

```diff
@@ -705,6 +705,7 @@ void service::merge(object const& obj) {
   service const& tmpl(static_cast<service const&>(obj));
 
   MRG_OPTION(_acknowledgement_timeout);
+  MRG_OPTION(_initial_state);
```

Comment:
are you sure we have to make this field inheritable?

Reply:
will fix, the fields will be non-configurable.